dxr/build.py

```diff
@@ -447,7 +447,7 @@ def index_file(tree, tree_indexers, path, es, index):
     # will not accept the path, so just move on.
     rel_path = relpath(path, tree.source_folder)
     try:
-        rel_path = rel_path.decode('utf-8')
+        rel_path = unicode(rel_path, 'utf-8')
```

Comment:
Hmm, looking at the common points, it seems that all the paths come from unignored at some point, so we could toss the decode there, and ignore anything that won't decode.